### ISSUE TITLE
Proton 5.0-10 RC testing

### ISSUE THREAD

**Issue by aeikum** [ValveSoftware/Proton#4070]
Hello. A new build has been pushed to the `next` branch in the Steam client. With this build, Death Stranding is now playable.

The game does require the latest Nvidia and AMD drivers. We've had success on Nvidia with Nvidia drivers 440.100 and 450.57, and on AMD with Mesa 20.1.3 with LLVM 10.0.0. If you are on AMD and experience a graphics error dialog on startup, please restart the Steam client once to ensure you have the latest Proton configuration settings for the game.

In the Steam client, the Proton 5.0 app should have a "next" beta branch in its Properties dialog, which you can choose to start testing the 5.0-10 release candidates. Note that the name of the build in the Steam Settings dialog will not be updated. I will post changes here when we push new builds. The source for the latest RC build is available on the proton_5.0-next branch in these repositories. This branch may receive forced updates.

In this issue, we are interested in problems that are new to the 5.0-10 RC builds, and in feedback about Death Stranding. If you find new problems in the "next" branch, please confirm that the problem does not occur in on the "default" branch before reporting it here.

**Comment by Plagman** [comment]
Known [Death Stranding] issue: changing window mode or resolution crashes the GPU (at least on radv).

**Comment by julienbenjamin** [comment]
I am experiencing a crash when trying to launch Death Stranding, with Nvidia drivers 440.100 on Debian (kernel version 5.4.0).

![Screenshot from 2020-07-16 01-54-22](https://user-images.githubusercontent.com/9112511/87611227-c6122d80-c707-11ea-89ab-2ac4070cbcf6.png)


[steam-1190460.log](https://github.com/ValveSoftware/Proton/files/4928574/steam-1190460.log)

**Comment by msmafra** [comment]
> I am experiencing a crash when trying to launch Death Stranding, with Nvidia drivers 440.100 on Debian (kernel version 5.4.0).
> 
> ![Screenshot from 2020-07-16 01-54-22](https://user-images.githubusercontent.com/9112511/87611227-c6122d80-c707-11ea-89ab-2ac4070cbcf6.png)
> 
> [steam-1190460.log](https://github.com/ValveSoftware/Proton/files/4928574/steam-1190460.log)

Same happened with me with Fedora 32, Nvidia drivers 440.100 and kernel 5.7.8. The log file became giant 3.7GB


[steam-specs.txt](https://github.com/ValveSoftware/Proton/files/4928919/steam-specs.txt)
![Captura de tela de 2020-07-15 22-50-14](https://user-images.githubusercontent.com/1005457/87618573-93106f80-c6f0-11ea-9740-b7d85657ef22.png)

**Comment by flacks** [comment]
Error as posted above on Arch Linux with kernel 5.7.8 and NVIDIA 450.57-2.

**Comment by katiesvt** [comment]
Hi all, I was able to get it running with latest Arch (5.7.8/450.57-2 on a 1080 Ti) and it appeared to be working well up through the second cinematic (the one that includes the "Prologue" title card) on default options (albeit 1440p@120Hz). However, I haven't been able to get as lucky since - subsequent runs (which involved me changing vsync settings on/off or FPS limit or quality settings) appear to make it later freeze my entire system during that same cinematic.

Resetting the quality options and resolution options to the defaults got me through that cinematic - but now I am seeing a lot of rocks and foliage flying in the air and generally in the wrong spots (some of them are able to be walked through, some not), even across multiple restarts and reboots. I haven't seen any reports about this issue yet, so I thought I would leave one here and I hope it helps.

Other than that, the performance has been great. MangoHud reports my GPU usage at 100% at 1440p with above-60 FPS which is not what I expected on day two at all!

Happy to provide more info if required.

**Comment by SevgiZ** [comment]
I wasn't sure if I should post it here and in the DS thread, but I wrote a post with with the issues I found + system information. I'll just link to the post.
https://github.com/ValveSoftware/Proton/issues/4069#issuecomment-659161459

**Comment by flacks** [comment]
I posted too soon. The second time around trying to begin the game, it indeed plays through the first two cinematics as @katiesvt posted above. Similarly, rocks and foliage are also out of place, even during the cinematics (e.g., protagonist goes "through" some rocks).

Have not yet experienced system freeze, and I've restarted the game from the beginning several times now. Also using a 1080 Ti at max settings at 1080p/75Hz, and launching DS with Feral's GameMode.

From the report by @SevgiZ , I've also experienced quite a bit of stuttering and drops in framerate every so often. The screenshots are essentially what I've come across as well.

**Comment by rbernon** [comment]
@jul-ben @msmafra This happens because of some missing dll registration. Did you previously tried to start the game with an older Proton version? You may then try again after removing the prefix in <steam library>/compatdata/1190460/pfx

**Comment by TheDiscordian** [comment]
So far Death Stranding seems to run fine, other than lighting oddities in this cave and by water. I haven't gotten very far yet, and I don't see these issues often.
![EntranceToCave](https://user-images.githubusercontent.com/43145244/87648755-e6e57d80-c71d-11ea-986f-0a039f78b2da.png)
![WelcomeToTheAcidCave](https://user-images.githubusercontent.com/43145244/87648769-e947d780-c71d-11ea-947f-54ba4325bc10.png)


Edit: A restart made lighting normal. I'm very impressed guys, good work!

Edit 2: Only imperfection I have on my end is some textures don't load correctly. The game also presents me with a giant screen of colours for several seconds before going black and launching. Here's the texture issue:

![image](https://user-images.githubusercontent.com/43145244/87651251-cec32d80-c720-11ea-97a4-c06b0cb302b6.png)

Edit 3: On my second launch I got an infinite loading screen after walking down from the cave, downstream, and reaching the dam.

Edit 4:

```
Video Card:
    Driver:  X.Org AMD Radeon (TM) RX 480 Graphics (POLARIS10, DRM 3.36.0, 5.6.15-1-MANJARO, LLVM 10.0.0)
    Driver Version:  4.6 (Compatibility Profile) Mesa 20.0.7
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x1002
    DeviceID:  0x67df
    Revision Not Detected
    Number of Monitors:  2
    Number of Logical Video Cards:  1
    Primary Display Resolution:  2560 x 1440
    Desktop Resolution: 4240 x 1777
    Primary Display Size: 62.99" x 35.43" (72.24" diag)
                                            160.0cm x 90.0cm (183.5cm diag)
    Primary VRAM: 8192 MB
```

The lighting issues do re-occur upon loading new areas. Infinite loading happens during the corpse cutscene as mentioned later in the thread, I haven't tried again to get further.

**Comment by goldbay** [comment]
It actually works somehow, good job!

I have had problems with flickering of terrain, stuttering, bad performance (30-40 fps) and frequent crashes. Furthest I have gotten is the car cutscene with the body. During that game crashes always. At some point I had to delete save game to get back into game through loading screen. 

Now it seems that for some reason the performance is little bit better (over 60fps with flickering etc.), but game still crashes often. This is log from a crash when running downhill out of the cave. Screen freezes every time, but music/cutscene sounds keep going. [steam-1190460.log](https://github.com/ValveSoftware/Proton/files/4930846/steam-1190460.log)

I'm running Pop!_OS 20.04 with Mesa 20.1.3 and LLVM 10.0.0
[system_info.txt](https://github.com/ValveSoftware/Proton/files/4931054/system_info.txt)

Edit: Added specs, removed most of the typos.

**Comment by aeikum** [comment]
Thanks all for your reports. If you are reporting issues, please always include your GPU model and graphics driver information (nvidia driver version, or mesa and llvm version for AMD). You can find this information in Steam in the Help->System Information dialog, under the Video Card "Driver" and "Driver version" fields. If you didn't include this information, it would be helpful to edit it into your previous posts.

**Comment by julienbenjamin** [comment]
> @jul-ben @msmafra This happens because of some missing dll registration. Did you previously tried to start the game with an older Proton version? You may then try again after removing the prefix in /compatdata/1190460/pfx

I did not make an extensive testing but the game works now ; indeed, like you suspected I tried to launch the game with the current stable version of Proton (5.0-9) before and forgot to remove the old prefix folder.

So, I remove it. Weird thing happened: Steam did not launch the init script before I restart the client. After a fresh prefix created and the script init done, it seems to work. I did not try past the menu for now.

I do not know if that helps, the log seemed sort of sane (just a few Wine `fixme` related to dx12 calls), do not hesitate to tell me if you need more details.

**Comment by alex9k1** [comment]
Works fine for me so far. Game crashed once during the opening cutscene (happened on RADV). Performance ain't great (40-45 FPS), but playable. Changing any graphics settings doesn't seem to have any effect at all on the performance. Performance also seems to be the same for either RADV or AMDVLK. On RADV however, the sky is sometimes artifacted. No graphical glitches (so far) with AMDVLK.

I'm on a AMD Radeon VII with Mesa 20.1.3 with ACO / AMDVLK 2020.Q3-1 and LLVM 10.0.0

**Comment by msmafra** [comment]
My Fedora 32 got some updates yesterday for MESA, wine and others. And I tested the game on kernel 5.6.17-fsync and 5.7.8. It open and I tested for 10min +-. Can't know for sure about performance because I play on my laptop core i15-7300HQ with a mobile GTX 1050 4GB and 16GB/RAM.
[dnf-distro-sync.txt](https://github.com/ValveSoftware/Proton/files/4933106/dnf-distro-sync.txt)

**Comment by nutta-git** [comment]
Flickering in _Monster Hunter World_ with Dx12 rendering enabled. 
Not reproducible with Proton.5.0-9

System Info: 
OS: Arch Linux 
Kernel: 5.7.8-arch1-1
Gpu: RX 5700 
Driver: OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.0-devel (git-293f251871)

**Comment by LoreRealm** [comment]
Death stranding recent update is now producing guranteed access violation errors IE: https://imgur.com/a/J96zpno 
OS: Arch LinuxKERNEL: 5.7.8-arch1-1CPU: AMD Ryzen 7 2700X Eight-CoreGPU: NVIDIA GeForce RTX 2080 TiGPU DRIVER: NVIDIA 450.57 RAM: 32 GB
[error.txt](https://github.com/ValveSoftware/Proton/files/4939313/error.txt)

**Comment by PastNullInfinity** [comment]
The V1.01 update is triggering an access violation for me too: [https://imgur.com/a/4Yeu1o9](https://imgur.com/a/4Yeu1o9)

- Kernel version: pop-os 5.4.0-7634-generic
- CPU: AMD Ryzen 3600
- GPU: GTX 1660 Super 
- Nvidia driver: 440.100

**Comment by msmafra** [comment]
> The V1.01 update is triggering an access violation for me too: https://imgur.com/a/4Yeu1o9
> 
> * Kernel version: pop-os 5.4.0-7634-generic
> * CPU: AMD Ryzen 3600
> * GPU: GTX 1660 Super
> * Nvidia driver: 440.100

And for me too.

- Distro: Fedora release 32 (Thirty Two) x86_64;
- Kernel: 5.7.8-200.fc32.x86_64;
- CPU: Intel i5-7300HQ (4) @ 3.500GHz;
- GPU: NVIDIA GeForce GTX 1050 Mobile;
- Nvidia driver version: 440.100
- Mesa: 20.1.3;
- wine-5.12 (Staging); if matters.

**Comment by SeraphMorningstar** [comment]
Getting the same access violation errors as yurgburg, PastNullInfinity and msmafra  
https://imgur.com/9RIyuqk

OS: Fedora release 32 (Thirty Two) x86_64 
Kernel: 5.7.8-200.fc32.x86_64
CPU: Intel i7-6700K (8) @ 4.200GHz
GPU1: NVIDIA GeForce GTX TITAN X (Pascal)
GPU2: NVIDIA GeForce GTX TITAN X (Pascal)
Nvidia driver version: 450.57 
Wine: wine-5.12 (Staging)

**Comment by TheDiscordian** [comment]
After updating to latest Death Stranding version (V1.01) results in a crash, Windows version at end of log file says "Windows 8" winecfg on the prefix indicated Windows 10, not sure if that's relevant. I know I don't have the proper Mesa version, my distro should have it soon, but it was "mostly" working before on my current version.

```
Video Card:
    Driver:  X.Org AMD Radeon (TM) RX 480 Graphics (POLARIS10, DRM 3.36.0, 5.6.15-1-MANJARO, LLVM 10.0.0)
    Driver Version:  4.6 (Compatibility Profile) Mesa 20.0.7
    OpenGL Version: 4.6
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x1002
    DeviceID:  0x67df
    Revision Not Detected
    Number of Monitors:  2
    Number of Logical Video Cards:  1
    Primary Display Resolution:  2560 x 1440
    Desktop Resolution: 4240 x 1777
    Primary Display Size: 62.99" x 35.43" (72.24" diag)
                                            160.0cm x 90.0cm (183.5cm diag)
    Primary VRAM: 8192 MB
```

![CrashAfterPatch2](https://user-images.githubusercontent.com/43145244/87835287-b9eab500-c85a-11ea-83df-72c0e6da2a9f.png)
[steam-1190460.log](https://github.com/ValveSoftware/Proton/files/4940597/steam-1190460.log)
[error.txt](https://github.com/ValveSoftware/Proton/files/4940616/error.txt)

**Comment by largewedge** [comment]
I've also got an access violation running Debian Sid (kernel 5.7.0-1) on a ryzen 5 3600 with a radeon RX 5600XT. I wasn't sure about the drivers, but under system information, I see that I have LLVM 10.0.0, and the driver version is listed as 4.6 (Compatibility Profile) Mesa 20.1.2.

<details><summary>Here are the contents of steamapps/common/Death\ Stranding/error.txt: </summary><p>

```
Build : dso 102/1901735 22:46 - Thu Jul 16 2020

Internal system error occured.

An Access Violation (C0000005h) has occurred in thread 'Background 3' at instruction location 00000001416A1C35h

CallStack : 3405644613

  0. 0x00007BCDB36E    NtRaiseException
  1. 0x00007BCD458A    raise_func_trampoline
  2. 0x0001416A1C35    ds, 0x416A1C35
  3. 0x00014173DBE5    ds, 0x4173DBE5
  4. 0x000141700A65    ds, 0x41700A65
  5. 0x0001417525F1    ds, 0x417525F1
  6. 0x0001487537DF    ds, 0x487537DF
  7. 0x00014183F59C    ds, 0x4183F59C
  8. 0x00014183A45D    ds, 0x4183A45D
  9. 0x000141838E4C    ds, 0x41838E4C
 10. 0x0001417B0B74    ds, 0x417B0B74
 11. 0x0001417B0C6D    ds, 0x417B0C6D
 12. 0x0001417B1C9B    ds, 0x417B1C9B
 13. 0x0001417AD08D    ds, 0x417AD08D
 14. 0x00007BCD6BDD    DbgUserBreakPoint
 15. 0x000000000000    
 16. 0x0001417ACF80    ds, 0x417ACF80
 17. 0x04E47A011680    
 18. 0x000000000000    


RAX = 84c503d0 RBX = 84c503d0 RCX = 4b8f4f8 RDX = ccc44720 RSI = 0 RDI = 84c60c00
R8 = 0 R9 = 0 R10 = 550000 R11 = 0 R12 = 85c50088 R13 = 7a1c4330
R14 = 0 R15 = 1 RIP = 416a1c35 RSP = 4b8f4d0 RBP = 4b8f5b9 EFL = 10202

Dx12-ErrorCode : 0x80004005
DeviceRemovedReason: 0x0
VMEM: Used 0.00 MB / Budget 6144.00 MB
VMEM: Video 6144.00 MB / Shared 24185.75 MB / System 0.00 MB
MEM: Used 34266.38 MB / Total 48371.50 MB


  0. 0x0001417C0068    ds, 0x417C0068
  1. 0x0001417C1F41    ds, 0x417C1F41
  2. 0x0001417C2256    ds, 0x417C2256
  3. 0x00007BC776AB    __wine_spec_unimplemented_stub
  4. 0x00007BCDAF72    RtlVirtualUnwind
  5. 0x00007BCDB36E    NtRaiseException
  6. 0x00007BCD458A    raise_func_trampoline
  7. 0x0001416A1C35    ds, 0x416A1C35
  8. 0x00014173DBE5    ds, 0x4173DBE5
  9. 0x000141700A65    ds, 0x41700A65
 10. 0x0001417525F1    ds, 0x417525F1
 11. 0x0001487537DF    ds, 0x487537DF
 12. 0x00014183F59C    ds, 0x4183F59C
 13. 0x00014183A45D    ds, 0x4183A45D
 14. 0x000141838E4C    ds, 0x41838E4C
 15. 0x0001417B0B74    ds, 0x417B0B74
 16. 0x0001417B0C6D    ds, 0x417B0C6D
 17. 0x0001417B1C9B    ds, 0x417B1C9B
 18. 0x0001417AD08D    ds, 0x417AD08D
 19. 0x00007BCD6BDD    DbgUserBreakPoint
 20. 0x000000000000    
 21. 0x0001417ACF80    ds, 0x417ACF80
 22. 0x04E47A011680    
 23. 0x000000000000    
000000  
```

</p></details>

The game itself launches but is stuck at a black screen.

**Comment by mikolfaro** [comment]
I got the same error on
- OS: PopOS 20.04
- Kernel: 5.4.0-7634
- CPU: AMD Ryzen 5 3600
- GPU: AMD 3600XT
- Mesa: 20.1.2
- LLVM: 9.0.1

The game crashes at launch without  black screen, just the popup.

[error.txt](https://github.com/ValveSoftware/Proton/files/4941942/error.txt)

**Comment by Plagman** [comment]
The "next" branch has now been updated with a vkd3d fix for the new compute shaders in 1.01. It appears to work on NVIDIA, but currently crashes on radv+ACO, pending a driver fix. Will post more info when we have it, but in the meantime NVIDIA users should be able to play the game again after getting the Proton update on the "next" branch.

**Update:** It also works on AMDGPU-PRO 20.20 on AMD GPUs. The current radv crash is caused by remaining vkd3d issues and will need additional Proton updates, will post an update next week when we know more.

**Comment by nimatel** [comment]
> The "next" branch has now been updated with a vkd3d fix for the new compute shaders in 1.01. It appears to work on NVIDIA, but currently crashes on radv+ACO, pending a driver fix. Will post more info when we have it, but in the meantime NVIDIA users should be able to play the game again after getting the Proton update on the "next" branch.

can confirm not working with radeon cards. Thanks for the speedy work fellas

**Comment by AngryPenguinPL** [comment]
> **Update:** It also works on AMDGPU-PRO 20.20 on AMD GPUs. The current radv crash is caused by remaining vkd3d issues and will need additional Proton updates, will post an update next week when we know more.

What with AMDVLK?

**Comment by Sil3ntPr0digy** [comment]
> > **Update:** It also works on AMDGPU-PRO 20.20 on AMD GPUs. The current radv crash is caused by remaining vkd3d issues and will need additional Proton updates, will post an update next week when we know more.
> 
> What with AMDVLK?

AMDVLK is far more stable.

**Comment by FeepingCreature** [comment]
Crashes with RADV and `RADV_DEBUG=llvm` too. So not ACO specific, far as I can tell.

**Comment by leillo1975** [comment]
I know that this fixes don't have relationship with Death Stranding, but could you include them in the following version?:
**Assetto Corsa Competizione:** https://github.com/wine-mirror/wine/commit/3078f10d43d834b0498358fe0accb565191b7020
https://github.com/wine-staging/wine-staging/commit/8402c959617111ac13a2025c3eb7c7156a2520f8
**rFactor 2:** https://github.com/wine-mirror/wine/commit/60c8c78015b12493eb3984feef29103724a758af

**Comment by valters-tomsons** [comment]
`Proton 5.0-next` seems to break Death Stranding `v1.00` and `v1.01` for me. Game never actually freezes, but after the main menu, loading screen gets stuck at 99% and `Now Loading...` icon doesn't stop spinning. 

Also tested without `esync` / `fsync` with no results.

This was not the case before https://github.com/ValveSoftware/Proton/issues/4070#issuecomment-663693971

[System information here.](https://gist.githubusercontent.com/FaithLV/0dfac08631613f97913fbe18058d4fae/raw/47d3035856d115e6a97b5255a120ace0ecd2282e/sysinfo)

**Comment by GloriousEggroll** [comment]
@FaithLV it takes quite a while to load, especially on a mechanical hdd even after the status bar is full (i experienced the same thing) after waiting a long while it finally loaded. I played about 2 hours last night on amdvlk with no issues other than long loading

**Comment by valters-tomsons** [comment]
@GloriousEggroll I've waited over 20 minutes with no signs of it ever finishing, also this was not the case prior to `vkd3d` update.

**Comment by leillo1975** [comment]
Take a look...
https://github.com/ValveSoftware/Proton/issues/1420#issuecomment-664282802

**Comment by Plagman** [comment]
The 'next' branch has just been updated with a fix for the crashes on startup with radv.

**Comment by nimatel** [comment]
> The 'next' branch has just been updated with a fix for the crashes on startup with radv.

Don't want to step on anyone's toes.. Game starts and crashes Access Violation 'Background 4'

popOS
ryzen 7
rx 480 8GB

**Comment by AdsDory** [comment]
Game now starts for me, however keep getting Random Freezes and have to exit the game.

Very new to Linux but happy to look into logs etc if people can point me in the right direction

Linux Mint 20
Ryzen 3600x
5700xt

**Comment by rodrigobbarros** [comment]
Hello someone can help me when I'm trying to open the game, It opens and the hangs on a black screen I'm using the next proton.
And I also tested it on the proton-5.9-ge-4-st
_'m in the_ ubuntu 20.04 lts 
cpu i7-7700hq
gpu gtx 1050ti 4gb vram driver 450.57
16gb ram
kernel 5.7.10

**Comment by largewedge** [comment]
> I've also got an access violation running Debian Sid (kernel 5.7.0-1) on a ryzen 5 3600 with a radeon RX 5600XT. I wasn't sure about the drivers, but under system information, I see that I have LLVM 10.0.0, and the driver version is listed as 4.6 (Compatibility Profile) Mesa 20.1.2.
Here are the contents of steamapps/common/Death\ Stranding/error.txt:

> The game itself launches but is stuck at a black screen.

Using the newest next update, I can launch and start a new game. It gets a few minutes into the opening scene (usually around when the motorcycle stops the first time, or as it reaches the cliff) before it freezes. The audio usually continues, however. Here's a log file:
[steam-1190460.log](https://github.com/ValveSoftware/Proton/files/4991897/steam-1190460.log)

Edit: I've tried with both LLVM and ACO, and each time I've also had mangohud enabled in the launch options. I also tried turning the graphics settings from the highest settings down to default, then I tried turning off vsync and capping the framerate at 60, butt none of these things make a difference.

**Comment by Semperverus** [comment]
Like the previous two posters, I am getting random freezes in Death Stranding as well. Freezes are guaranteed in the middle of, or towards the end of cutscenes. I also continue to hear audio play through the cutscenes if it crashes during them most of the time. Other crashes seem to be instigated when opening menus (though this is not consistent) and when transitioning from indoors to outdoors areas.

Also, I am getting super pixelated shadows of objects in the sky that have extreme parallaxing when I turn the camera. It looks very odd to have a sky that looks like an Atari 2600 vomited all over it.

My setup:

Software
Arch Linux
Kernel 5.7.10
Mesa 20.1.4 (amdgpu)
RADV/ACO

Hardware
AMD Ryzen 1700X (3400Mhz)
AMD/XFX 5700xt (thicc III Ultra, "standard" mode on dipswitch)
ASRock Taichi x370
16gb G.Skill Trident RGB (3000Mhz, 2x8GB)
Samsung 970 EVO NVMe 1TB

**Comment by nutta-git** [comment]
> Flickering in _Monster Hunter World_ with Dx12 rendering enabled.
> Not reproducible with Proton.5.0-9
> 
> System Info:
> OS: Arch Linux
> Kernel: 5.7.8-arch1-1
> Gpu: RX 5700
> Driver: OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.0-devel (git-293f251871)

Issue is still present with current RC 
Kernel: 5.7.10-arch1-1
Driver: OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.0-devel (git-18fe130ec9)

**Comment by jhaand** [comment]
It remains awfully quiet here, while I'm still unable to run the game. 
See: https://github.com/ValveSoftware/Proton/issues/4069#issuecomment-666202990

How can I get better tracinng on what's crashing?
Where can I find the development of Proton 5.0 RC5? 
And where is the development discussion about getting Death Stranding working correctly?

**Comment by Vladislavuz** [comment]
What about SteamOS?

**Comment by ghost** [comment]
> Where can I find the development of Proton 5.0 RC5?

Quite sure Valve keeps certain things to themselves. You can find the 5.0-next branch on here, but for example, the last fix said above won't be found in it until Valve updates it with their local changes (or unless its pulled in from another project but there's no rc5 in that case). I don't think they have it stashed in another public github project.

So you can basically just continue to look here. If you want the gritty details then look at the branch for whatever Valve has made public.

**Comment by ghost** [comment]
There is massive artifacting in Death Stranding like rocks spawning in the air and such, though I don't get [this](https://github.com/ValveSoftware/Proton/issues/4069#issuecomment-668086387) issue. I already mentioned this issue [here](https://github.com/ValveSoftware/Proton/issues/4069#issuecomment-674321775), but I will also post here just in case if other people don't see my other comment. I am also running the "next" build to even get Death Stranding running.

<details><summary>Some images of the artifacting:</summary><p>

![Screenshot from 2020-08-14 19-27-08](https://user-images.githubusercontent.com/21233238/90301288-9b310b80-de64-11ea-8fe0-fe6ec5143960.png)
![Screenshot from 2020-08-14 19-27-32](https://user-images.githubusercontent.com/21233238/90301289-9cfacf00-de64-11ea-8e29-11e8f20d304e.png)
![Screenshot from 2020-08-14 19-27-49](https://user-images.githubusercontent.com/21233238/90301290-9e2bfc00-de64-11ea-9028-79d37890376a.png)
![Screenshot from 2020-08-14 19-28-26](https://user-images.githubusercontent.com/21233238/90301292-9ff5bf80-de64-11ea-8b73-05dd768f5ac6.png)
![Screenshot from 2020-08-14 19-29-02](https://user-images.githubusercontent.com/21233238/90301294-a08e5600-de64-11ea-9e0e-1c613d1a2a79.png)

</p></details>

I would also like to point out that if I launch Death Stranding via big picture mode, the artifacting issue almost completely fixes its self. From time to time it would artifact again, but that is easily solvable by restarting the game via big picture.

**Comment by Dreamurrrr** [comment]
Some pictures of this artifacting in Death Stranding:
![Screenshot from 2020-08-14 19-27-08](https://user-images.githubusercontent.com/21233238/90301288-9b310b80-de64-11ea-8fe0-fe6ec5143960.png)
![Screenshot from 2020-08-14 19-27-32](https://user-images.githubusercontent.com/21233238/90301289-9cfacf00-de64-11ea-8e29-11e8f20d304e.png)
![Screenshot from 2020-08-14 19-27-49](https://user-images.githubusercontent.com/21233238/90301290-9e2bfc00-de64-11ea-9028-79d37890376a.png)
![Screenshot from 2020-08-14 19-28-26](https://user-images.githubusercontent.com/21233238/90301292-9ff5bf80-de64-11ea-8b73-05dd768f5ac6.png)
![Screenshot from 2020-08-14 19-29-02](https://user-images.githubusercontent.com/21233238/90301294-a08e5600-de64-11ea-9e0e-1c613d1a2a79.png)

**Comment by tokc** [comment]
I'll report the issues I am experiencing, even though I see I am not the only one with similar symptoms. Perhaps the extra data point will be helpful in diagnosing it and creating a solution.
If nothing else, then to add another voice to "patiently waiting for a solution!"

## Technical Information
### Summary
Game: Death Stranding
Date: 2020.08.17
GPU: AMD Radeon RX 5700 XT, LLVM 10.0.1, Mesa 20.1.5
Proton: 1595878477 proton-5.0-10-rc4
SteamGameId: 1190460

### Full details
[ATTEMPT_A_steam-1190460.log](https://gist.github.com/tokc/3d71d42cbff5f194f4e2d3acbb78c7ca)
[ATTEMPT_B_steam-1190460.log](https://gist.github.com/tokc/840b02fa042833cd3c5bbe61a769f19e)
[system_information.txt](https://gist.github.com/tokc/e44d5a31282e3bba34bdaf3667d042e2)

## Symptoms
- Game visuals and/or screen becomes frozen in varying states of the game. For example: During a loading screen. During a cinematic. Immediately after a cinematic but before gameplay resumes. During gameplay.
- Audio continues to play: When the freeze occurs during a cinematic, the rest of the audio track of the cinematic plays out. When the freeze occurs during gameplay, any continuous ambient noises continue to play while the screen is frozen.

## Reproduction
I have not observed a pattern to when or where the freeze occurs. Sometimes it happens within seconds, sometimes the game runs normally for a few minutes before freezing.
Issue does not occur in Proton 5.0-9, as the game does not launch at all using 5.0-9

-------------------------------------

## Additional Information
Earlier attempts to run the game resulted in freezes during the initial several cinematics of the start of the game.
I have included two Proton log files (see links above) from two of my attempts to run the game.
The following is a description of what happened during those two attempts:

ATTEMPT_A:
- Windowed mode
- Initial loading screen worked fine.
- Initial menuing (date of birth, brightness, etc.) worked fine.
- Unskippable Kojipro cinematic played fine.
- Was able to skip the two intro cinematics.
- Was able to save.
- Game was playable for a minute or two.
- Screen froze, desktop environment unresponsive.
- Killed ds.exe by sending SIGTERM through htop in another session.

ATTEMPT_B:
- Change from windowed mode to fullscreen.
- Increase memory for streaming to "High".
- Was able to continue from the save I made at the very start of the game.
- Game was playable for a minute or two.
- Screen froze, desktop environment unresponsive.
- Killed ds.exe by sending SIGTERM through htop in another session.

Graphical settings mostly lowest / off.

During the few minutes when I was able to control the game, everything seemed to look and feel as expected.
I was briefly able to run around the starting area, pick up some cargo, and during one of the attempts I even fell off the cliff and went through the entire repatriating sequence without any noticeable issues. So I would hypothesize that the issue is not related to a particular part of the game or in-game action.
No noticeable performance issues, although I've only been able to play for one to two minutes at a time, in the very first area.

## Other Notes
I just want to thank everyone involved for their patience both in trying the game and in patching Proton, so that everyone may some day enjoy this wonderful game running with stability on Linux.

**Comment by nutta-git** [comment]
> > Flickering in _Monster Hunter World_ with Dx12 rendering enabled.
> > Not reproducible with Proton.5.0-9
> > System Info:
> > OS: Arch Linux
> > Kernel: 5.7.8-arch1-1
> > Gpu: RX 5700
> > Driver: OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.0-devel (git-293f251871)
> 
> Issue is still present with current RC
> Kernel: 5.7.10-arch1-1
> Driver: OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.2.0-devel (git-18fe130ec9)

Issue is no longer present 
OS: Arch Linux
Kernel: 5.8.7-zen1-1-zen
Gpu: RX 5700
Driver: mesa-tkg-git 20.3.0_devel.128067.4b7f93b4079-1

**Comment by hjri** [comment]
3.5 hours into Death Stranding, no crashes no major glitches, no tweaking required. :eyes: ~except that i had to wipe my prefix because it was created with proton 4.11 and used different version of windows~

The only minor glitch that i see is that some characters have seams/weird lines on them:  
![image](https://user-images.githubusercontent.com/3514245/92422977-016d2d80-f188-11ea-9b73-a16b3470eca6.png)

This happens on certain characters nearly constantly and on some only very up close.

With my Ryzen 9 3950X and 5700 XT performance on 1080 is stellar - around 100fps, on 1440p drops to around 50 which is acceptable. All graphical settings are at "default". It does drop/freeze on scene changes however but not enough to bother me at least.

**Comment by TiZ-HugLife** [comment]
What are the changes in 5.0-10? Anything DRM-related, that might help with #1780?

**Comment by kisak-valve** [comment]
Hello @HugLifeTiZ, the key details are in the opening post at the top of this issue report. You can look through the finer details at https://github.com/ValveSoftware/Proton/tree/proton_5.0-next and the various sub-components.

**Comment by TiZ-HugLife** [comment]
The OP only mentions that it made Death Stranding playable; not what was changed in order to do that. I didn't know if it was only DX12 stuff or if anything potentially relating to DRM needed to be changed too. But still, that link you included was pretty helpful. Looks like there are a few Wine changes that might be relevant, so it'll be worth putting on the todo list once my stupid Denuvo timer runs out. Thank you.

**Comment by nutta-git** [comment]
> The OP only mentions that it made Death Stranding playable; not what was changed in order to do that

https://github.com/ValveSoftware/wine/commit/e9264df6e63b5df87d81e950675df7290ad43615
https://github.com/ValveSoftware/wine/commit/6085222f76135e10abefdd5f58d3fdcf86a904c8

**Comment by Galcian79** [comment]
Just a little reminder (#4186) to merge the latest xinput patches to next official proton release.
Games that would benefit:

Mass Effect: Andromeda controller support
Crysis 3 controller support
Maybe other games

**Comment by vjr2** [comment]
Can you also include the alyx fixes in the next 5.0-10 version? Path of Exile now only runs with the old 5.0-7-alyx2 build, and that's frustrating

**Comment by aeikum** [comment]
Hello, all. Thanks for testing the 5.0-10 RC. All of the changes from that RC have been included in the new 5.13 release, so we are not going to release this RC on the 5.0 branch. I'll be removing that branch and closing this issue in the very near future.
